eval/eval.go

```diff
@@ -304,7 +304,7 @@ func (s *State) applyFunction(fn object.Object, args []object.Object) object.Obj
 	}
 	curState := s.env
 	s.env = nenv
-	res := s.evalInternal(function.Body)
+	res := s.Eval(function.Body) // Need to have the return value unwrapped. Bug #46
```

Comment:
Either it's a todo


```suggestion
	res := s.Eval(function.Body) // TODONeed to have the return value unwrapped. Bug #46
```

Either is was fixed and I would expect to do not mention "bug"

```suggestion
	res := s.Eval(function.Body) // Need to have the return value unwrapped. See #46
```

Reply:
it _fixes_ Bug #46, I can add Fixes #46, there is no todo it's the fix

Reply:
fixed in https://github.com/grol-io/grol/pull/53/commits/4097534d4e86fab40035cb38c8cf087d598da88d